test/unit/html.pre.test.js

```diff
@@ -34,59 +34,6 @@ describe('Testing pre requirements for main function', () => {
 });
 
 describe('Testing pre.js', () => {
-  it('Body content is wrapped in a div', () => {
-    const dom = new JSDOM('<html><head><title>Foo</title></head><body><h1>Title</h1></body></html>');
-    const context = {
-      content: {
-        document: dom.window.document,
-      },
-      request,
-    };
-    pre(context);
-
-    const div = dom.window.document.querySelector('div');
-    assert.ok('A div must have been added', div);
-    assert.equal(div.innerHTML, '<h1>Title</h1>');
-    assert.ok(div.classList.contains('default'));
-  });
-
-  it('Div is wrapped with class name', () => {
-    const dom = new JSDOM('<html><head><title>Foo</title></head><body><h1>Title</h1></body></html>');
-    const context = {
-      content: {
-        document: dom.window.document,
-        meta: {
-          class: 'customcssclass',
-        },
-      },
-      request,
-    };
-    pre(context);
-
-    const div = dom.window.document.querySelector('div');
-    assert.ok(div.classList.contains('default'));
-    assert.ok(div.classList.contains('customcssclass'));
-  });
-
-  it('Div is wrapped with multiple class names', () => {
-    const dom = new JSDOM('<html><head><title>Foo</title></head><body><h1>Title</h1></body></html>');
-    const context = {
-      content: {
-        document: dom.window.document,
-        meta: {
-          class: 'customcssclass, customcssclass2',
-        },
-      },
-      request,
-    };
-    pre(context);
-
-    const div = dom.window.document.querySelector('div');
-    assert.ok(div.classList.contains('default'));
-    assert.ok(div.classList.contains('customcssclass'));
-    assert.ok(div.classList.contains('customcssclass2'));
```

Comment:
While removing the `default` class is the result of this PR, I think the `customcssclass` or  `customcssclass2` should still be there to be able to "type" a page or a section.